app/javascript/.storybook/main.js

```diff
@@ -80,4 +81,8 @@ module.exports = {
 
     return config;
   },
+  babel: async (options) => ({
+    ...options,
+    plugins: [...options.plugins, 'inline-react-svg'],
```

Comment:
Storybook has its own Babel configuration, so we need to add our changes here as well.